apps/namadillo/src/atoms/integrations/functions.ts

```diff
@@ -168,21 +158,17 @@ const findCounterpartChainName = (
 
 const tryCoinToIbcAsset = async (
   coin: Coin,
-  rpc: string,
+  ibcAddressToDenomTrace: (address: string) => Promise<DenomTrace | undefined>,
   chainName: string
-): Promise<AssetAndIbc | undefined> => {
+): Promise<Asset | undefined> => {
   const { denom } = coin;
 
-  if (!denom.startsWith("ibc/")) {
+  const denomTrace = await ibcAddressToDenomTrace(denom);
+  if (typeof denomTrace === "undefined") {
```

Comment:
cosmetic, we can use `if (denomTrace === undefined)` without the `typeof` :)

Reply:
I started doing this based on a previous PR comment, but I think the difference in behaviour is pretty minor:
https://github.com/anoma/namada-interface/pull/619#discussion_r1481013086